core/src/edu/cornell/gdiac/temporary/entity/Note.java

```diff
@@ -27,172 +20,187 @@
 	/** The number of animation frames in our filmstrip */
 	private static final int   NUM_ANIM_FRAMES = 4;
-
-	/** Friction multiplier for this shell */
-	//private float friction;
-	/** Minimum Y velocity for this sheall */
-	private float minvelocy;
 	/** Current animation frame for this shell */
 	private float animeframe;
-	
-	/** To measure if we are damaged */
-	private boolean damaged;
-	/** The backup texture to use if we are damaged */
-	private Texture dmgTexture;
 
-	public int hitStatus;
+	private int hitStatus;
+
+	public int getHitStatus(){
+		return hitStatus;
+	}
+	public void setHitStatus(int t){
+		hitStatus = t;
+	}
 
 	/** line the note is one */
-	public int line;
+	private int line;
+	public int getLine(){
+		return line;
+	}
+	public void setLine(int t){
+		line = t;
+	}
+	private int startFrame;
+	public int getStartFrame(){
+		return startFrame;
+	}
+	public void setStartFrame(int t){
+		startFrame = t;
+	}
+	private int holdFrame;
 
-	public int startFrame;
-	public int holdFrame;
+	public int getHoldFrames(){
+		return holdFrame;
+	}
+	public void setHoldFrames(int t){
+		holdFrame = t;
+	}
 
 	public enum NType{
 		SWITCH,
 		HELD,
 		BEAT
 	}
 
-	public NType nt;
+	private NType nt;
 
-	/**
-	 * Returns the type of this object.
-	 *
-	 * We use this instead of runtime-typing for performance reasons.
-	 *
-	 * @return the type of this object.
-	 */
-	public ObjectType getType() {
-		return ObjectType.NOTE;
+	public NType getNoteType(){
+		return nt;
+	}
+	public void setNoteType(NType t){
+		nt = t;
 	}
+	private float w;
 
+	public float getWidth(){
+		return w;
+	}
+	public void setWidth(float t){
+		w = t;
+	}
+	private float h;
 
-	public float getRadius(){
-		return super.getRadius() * SHELL_SIZE_MULTIPLE;
+	public float getHeight(){
+		return h;
 	}
-	
-	/**
-	 * Sets whether this shell is destroyed.
-	 *
-	 * Shells have to be shot twice to be destroyed.  This getter checks whether this 
-	 * shell should be destroyed or it should just change colors.
-	 *
-	 * @param value whether this shell is destroyed
-	 */
-	public void setDestroyed(boolean value) {
-		destroyed = true;
+	public void setHeight(float t){
+		h = t;
+	}
+	private float x;
+
+	public float getX(){
+		return x;
+	}
+	public void setX(float t){
+		x = t;
 	}
-	
+	private float y;
+	public float getY(){
+		return y;
+	}
+	public void setY(float t){
+		y = t;
+	}
+	private float vy;
+	public float getYVel(){
+		return vy;
+	}
+	public void setYVel(float t){
+		vy = t;
+	}
+	private float by;
+	public float getBottomY(){
+		return by;
+	}
+	public void setBottomY(float y){
+		by = y;
+	}
+	private boolean destroyed;
+	public boolean isDestroyed(){
+		return destroyed;
+	}
+	public void setDestroyed(boolean d){
+		destroyed = d;
+	}
+	FilmStrip animator;
+	Vector2 origin;
+
 	/**
 	 * Initialize shell with trivial starting position.
 	 */
-	public Note(int line, NType n) {
+	public Note(int line, NType n, int frame) {
```

Comment:
note texture should be initialized here imo

Reply:
Noted, will change